### DISCUSSION TITLE
'GoLang Early access'

### DISCUSSION THREAD

**Discussion by lisaconnect1234** [prisma/prisma-client-go#524] [Q&A] [answered]
## Problem

GoLang language is in early access, what is the plan for full support on roadmap.  We are looking to use Golang with Prisma, what are the risks of using that in production now and will you be fully supporting in the future.

**Comment by lisaconnect1234** [comment]
Hi, I wanted to confirm if you intend to support Golang and if you would recommend using the Golang Prima in production?

**Comment by steebchen** [comment, marked as answer]
See https://github.com/steebchen/prisma-client-go/issues/860 for an elaborate, up-to-date answer for this.
